modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-2419[OSSM-2419] Istiod pod starts but gets stuck in not ready state due to failed probes. Sometimes, when a mesh contains many member namespaces, the Istiod pod does not become ready due to a deadlock within Istiod. The deadlock has been resolved and the pod now starts as expected.
```

Comment:
Use the past tense to describe the previous broken behavior and the present tense to describe the current fixed behavior.

Reply:
Some of the following release notes don't follow the before/after past/present formula consistently, creating some ambiguity.

Reply:
done